timesketch/lib/stories/api_fetcher.py

```diff
@@ -165,6 +173,31 @@ def get_aggregation_group(self, agg_dict):
         }
         return data
 
+    def get_indices_and_timelines(self, index_list):
+        """Returns a tuple with two lists from indices and timeline IDs.
+
+        Args:
+            index_list (list): A list of timeline IDs (int) and
+                indices (str).
+
+        Returns:
+            A tuple with two items, a list of indices and a list of
+            timeline IDs.
+        """
+        indices = []
+        timeline_ids = []
+
+        if isinstance(index_list, str):
+            index_list = index_list.split(',')
+
+        for item in index_list:
```

Comment:
`item` is a bit generic here, can you rename to `index` ?

Reply:
yes, changed it